code/modules/projectiles/guns/energy/stun.dm

```diff
@@ -54,6 +54,11 @@
 	flight_y_offset = 10
 	can_holster = TRUE
 
+/obj/item/gun/energy/disabler/Initialize(mapload, ...)
```

Comment:
```suggestion
/obj/item/gun/energy/disabler/Initialize(mapload)
```
additional args aren't even used by this type or it's parents, can just remove. Otherwise code is good